src/test/java/array/Array2dTest.java

```diff
@@ -0,0 +1,209 @@
+package array;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
```

Comment:
so, my comments will be the same as for a file above

Reply:
commented and fixed above

Reply:
thanks!